### PR TITLE
docs(v2): introduction

### PR DESCRIPTION
**Pull request by endiliey**
# Motivation

Add docs for introduction and comparison with other tools. Please help to improve it

cc @wgao19 as well

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/Docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

- Local Development
image

- Netlify
https://deploy-preview-1501--docusaurus-2.netlify.com/docs/introduction

### CODE REVIEW

File: website/docs/introduction.md [hunk 1]
```diff
@@ -3,33 +3,69 @@ id: introduction
...
+## GitBook
+
+The primary problem with GitBook is that the team behind it is more focused on turning it into a commercial product rather than an open-source tool. It also has a lot of problems that Docusaurus doesn't have. That's why [Redux switched to Docusaurus](https://github.com/reduxjs/redux/issues/3161). It is only free for open-source and & non-profit teams. Docusaurus is free for everyone.
+
+## Gatsby
+
+Gatsby is capable of doing what Docusaurus does. But it is designed to build various kind of websites and apps. In comparison, Docusaurus is very focused on content-centric sites and provides features tailed for documentation of the box. 
+Gatsby tries to **do many things well**, while Docusaurus tries to **do one thing super well**.
+
+In addition, Gatsby's codebase is complex and it has hundreds of open issues. In comparison, Docusaurus codebase is very minimal and it has less open issues. It is much harder to fix a bug caused by Gatsby yourself unless you have a vast understanding of Gatsby's monorepository which has has ~~9000~~ over 100 packages. You'll have to rely on the maintainers who have few other hundreds of issues to solve.
```
Comment (yangshun) [inline website/docs/introduction.md]: Remove this line. This sounds like we're speaking badly about Gatsby. Large, mature and popular projects will inevitably end up in that state. Take [React](https://github.com/facebook/react) for example.

File: website/docs/introduction.md [hunk 2]
```diff
@@ -3,33 +3,69 @@ id: introduction
...
+
+In addition, in order to achieve what Docusaurus does in Gatsby, you'll have to invest a lot of effort and build everything yourself. This of course give you an advantage of flexibility because you will be doing mostly everything on your own. However, that just means you're building another version of Docusaurus. Why reinvent the wheel ?
```
Comment (yangshun) [inline website/docs/introduction.md]: This too. Gatsby people aren't gonna be happy reading this, and this will not be true when they launch themes.

Comment (endiliey) [reply website/docs/introduction.md]: Sure thing !!

This is roughly just what came to my mind. Definitely need an edit.
Maybe I should add **Even if there is a Gatsby Theme that achieve that, that just means someone is building "another version of Docusaurus" again**

**Comment by endiliey** [comment]
Sure thing, please revise the content. I'll let you merge it. Just sending this PR before I go to sleep @yangshun

File: website/docs/introduction.md (hunk 1 above)
Comment (endiliey) [reply website/docs/introduction.md]: done

**Pull request merged**